src/Control/Monad/Eff/JQuery.purs

```diff
@@ -63,6 +118,24 @@ foreign import setAttr
   -> JQuery
   -> Eff (dom :: DOM | eff) Unit
 
+foreign import getAttrImpl
+  :: forall eff
+   . String
+  -> JQuery
+  -> Eff (dom :: DOM | eff) Foreign
+
+-- | Get an attribute value.
+getAttr 
+  :: forall eff
+   . String
+   -> JQuery
+   -> Eff (dom :: DOM | eff) (Maybe String)
+getAttr str jq = map foreignToString (getAttrImpl str jq)
+  where foreignToString f =
+          if isUndefined f
+            then Nothing
+            else Just (unsafeFromForeign f)
```

Comment:
Can it return Maybe String? or is it a nattrans from F to Maybe? I don't think the error can be useful, it will only inform the value was undefined.